docs/commands.md

```diff
@@ -0,0 +1,146 @@
+## Actions
+
+| Command                                          | Description                                                                                     |
+|--------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| `.add <i>pickup</i>[ <i>pickup</i> ..]] or +<u>pickup</u>[ <i>pickup</i> ..]]` | Adds you to specified pickups.                                                                |
```

Comment:
I would like to remove the `<i></i>` tags as they don't work in inlined code. Also, put aliases on each line. I can do this after merged if you're not up to it.